docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -41,13 +41,12 @@ Check services' status:
 ### Connect to Service
 
 ```sh
-> bin/nebula -u=user -p=password --addr={graphd IP address} --port={graphd listening port}
```

Comment:
The client needs to specify the port and IP of the graphd, when the client and the requested graphd are on the same host it can without these params

Reply:
done